include/psa/crypto_config.h

```diff
@@ -1423,25 +1375,26 @@
  *
  * \note Valid values: 128 or default of 256.
  */
-#define MBEDTLS_PSA_CRYPTO_RNG_STRENGTH                 256
+//#define MBEDTLS_PSA_CRYPTO_RNG_STRENGTH               256
 
 /**
  * \def MBEDTLS_PSA_CRYPTO_RNG_HASH
  *
  * \brief Hash algorithm to use for the entropy module and for HMAC_DRBG if configured.
  *
- * If the option is disabled, it is automatically defined in
- * crypto_adjust_config_derived.h based on:
- * - Whether MBEDTLS_PSA_CRYPTO_C, MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG,
- * PSA_WANT_ALG_SHA_256, PSA_WANT_ALG_SHA_512, MBEDTLS_CTR_DRBG_C,
- * and MBEDTLS_HMAC_DRBG_C are defined
- * - The value of MBEDTLS_PSA_CRYPTO_RNG_STRENGTH.
+ * The hash size (in bits) must be at least #MBEDTLS_PSA_CRYPTO_RNG_STRENGTH.
+ *
+ * In addition, if the entropy module is enabled (#MBEDTLS_PSA_CRYPTO_C is enabled
+ * and #MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG is disabled):
+ * - The hash size must be at least 32 bytes (i.e., 256 bits).
```

Comment:
I don't disagree with this bulletpoint, but out of curiosity, why are we requiring this?